tegra/tegra.c

```diff
@@ -206,7 +206,7 @@ int drm_tegra_bo_unref(struct drm_tegra_bo *bo)
 		return -EINVAL;
 
 	if (!atomic_dec_and_test(&bo->ref))
-		return 0;
+		return atomic_read(&bo->ref);
```

Comment:
Hmm, I don't see any configure options for a debug build. Could you elaborate, please?